languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/style.xml

```diff
@@ -13001,7 +12986,7 @@ USA
             <example correction="affecter"><marker>assigner</marker> à une tâche</example>
             <example correction="affecté">Il a été <marker>assigné</marker> à une tâche.</example>
         </rule>
-        <rule id="AMI_DE_GARCON" name="ami de garçon/gars" tone_tags="clarity">
+        <rule id="AMI_DE_GARCON" name="ami de garçon/gars" tone_tags="general">
```

Comment:
ahaha wow, cute, this rule has only 11 opens ever :D, AMI_DE_FILLE just below reaches 61, strangely. The rule was apparently created in march, 2022, but the originam file (grammar.xml) is too large to be displayed in github blame mode, sadly. Guess we will never know who created it ^^'

Reply:
It's the second time I go through this rule, it made me smile on both occasions hahaha I guess some L2 users can make that mistake but it's very very rare according to Grafana